packages/parse-argv/parse-argv.0.0.1/opam

```diff
@@ -21,4 +21,4 @@ depends: [
   "ounit"     {test}
   "astring"
 ]
-available: [ocaml-version >= "4.02.3"]
+available: [ocaml-version >= "4.03"]
```

Comment:
I've replied to the (old) thread about this on mirageos-devel, but note that unless there's a `mirage-bootvar-xen` release which depends on this package rather than its own implementation, the change affects only the MirageOS 3 universe.